src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs

```diff
@@ -174,6 +174,13 @@ public override void ViewDidLoad()
 			EnsureLayoutInitialized();
 		}
 
+		public override void LoadView()
+		{
+			base.LoadView(); 
+
+			CollectionView = new MauiCollectionView(CGRect.Empty, ItemsViewLayout);
```

Comment:
The `MauiCollectionView` type is in Core, but the usage is in Controls... Should we just keep it all in Cntrols? I am unfamilliar with this `CollectionView` property. Is it set in the core UIKit framework, or are we setting it somewhere and overriding it here. Not sure if we should just update the initial code to use this one and not override in Controls?

Reply:
[It is core UIKit property](https://developer.apple.com/documentation/uikit/uicollectionviewcontroller/1623983-collectionview?language=objc)

Reply:
OK, do we set it anywhere else? Also, maybe we can move the `MauiCollectionView` type into Controls/into this class.

Reply:
No, this is the only place. I am going to move it to Microsoft.Maui.Controls.Handlers.Items iOS folder and make it internal as discussed before